src/main/java/uk/gov/hmcts/reform/em/stitching/rest/WelcomeResource.java

```diff
@@ -31,13 +31,16 @@ public class WelcomeResource {
         produces = APPLICATION_JSON_VALUE
     )
     @ResponseBody
-    public ResponseEntity<String> welcome() {
+    public ResponseEntity<Object> welcome() {
 
         log.info("Welcome message : '{}'", MESSAGE);
 
+        Map<String, String> mapMessage = new HashMap<>();
+        mapMessage.put("message", MESSAGE);
+
         return ResponseEntity
-            .ok()
-            .cacheControl(CacheControl.noCache())
-            .body(MESSAGE);
+                .ok()
+                .cacheControl(CacheControl.noCache())
+                .body(mapMessage);
```

Comment:
just this would be enough
`Map.of("message",MESSAGE)`

Reply:
👍